src/components/CanvasEditor/Footer/Help/Help.scss

```diff
@@ -33,21 +34,41 @@
   }
 
   &__shortcuts {
-    border: 1px solid var(--light-border-color);
-    border-radius: var(--default-border-radius);
+    display: flex;
+    flex-direction: column;
 
     &-shortcut {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
       border-bottom: 1px solid var(--light-border-color);
 
       &:last-child {
         border-bottom: none;
       }
     }
 
+    &-content {
+      display: flex;
+      align-items: center;
+      width: 100%;
+
```

Comment:
Remove this spacing too, and Can u format scss file once I think its not formatted